.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Thank you for submitting the pull request to the Apache Ignite.
+
+In order to streamline the review of the contribution 
+we ask you to ensure the following steps have been taken:
+
+### The Contribution Checklist
+- [ ] There is a single JIRA ticket related to the pull request. 
+- [ ] The web-link to the pull request attached to the JIRA ticket.
+- [ ] The JIRA ticket has the _Patch Available_ state.
+- [ ] The pull request body describes changes that have been made. 
+The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
+- [ ] The pull request title treated as the final commit message. 
+The following pattern must be used: `IGNITE-12407: Java thin client. Cluster API support`
+- [ ] A reviewer has been mentioned through the JIRA comments 
+(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
+- [ ] The pull request has been checked by the Teamcity Bot and 
+the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))
+
+### Notes
+- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
+- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
+- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
+- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)
+
+If you need any help, please email to dev@ignite.apache.org or asking anу advice on http://asf.slack.com _#ignite_ channel.
```

Comment:
* `email to` -> `email` (it is a verb)
* `asking` -> `ask`

Reply:
Fixed.